/jds/bin/diff -uprN planner-0.14.2.old/data/planner.1 planner-0.14.2/data/planner.1
--- planner-0.14.2.old/data/planner.1	2009-02-20 16:40:12.648905000 +0000
+++ planner-0.14.2/data/planner.1	2009-02-20 16:46:48.412026000 +0000
@@ -1,3 +1,4 @@
+'\" te
 .TH "PLANNER" "1" "April 15th 2005" "Martin-Eric Racine" "GNOME Office"
 .SH "NAME"
 planner \- Project Management application for GNOME
@@ -16,6 +17,21 @@ with long options starting with two dash
 .TP 
 .B \-?, \-\-help
 Show summary of options.
+
+.SH "ATTRIBUTES"
+.PP
+See \fBattributes\fR(5)
+for descriptions of the following attributes:
+.sp
+.TS
+tab(^G) allbox;
+cw(2.750000i)| cw(2.750000i)
+lw(2.750000i)| lw(2.750000i).
+ATTRIBUTE TYPE^ATTRIBUTE VALUE
+Availability^SUNWplanner
+Interface stability^Uncommited
+.TE
+.sp
 .SH "SEE ALSO"
 .B planner
 offers a complete 
@@ -32,3 +48,4 @@ Kurt Maute <kurt@maute\.us>,
 along with several other contributors.
 .PP 
 This manual page was written by Martin\-Eric Racine <q\-funk@iki\.fi>.
+...\" LSARC 2008/454 Planner: project management tool for the Gnome desktop
